cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkLoginThroughInstitutions.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
To CR: for the new `action-state: institutionLogin`, triggered by `Event(institution)` in `action-state: remoteAuthenticate` and go on to the new `view-state: viewInstitutionLoginForm`.